dataland-frontend/src/components/resources/frameworkDataSearch/sme/SmeDataModel.ts

```diff
@@ -0,0 +1,371 @@
+import { SmeData } from "@clients/backend";
+import { Category } from "@/utils/GenericFrameworkTypes";
+
+export const smeDataModel = [
+  {
+    name: "general",
+    label: "General",
+    color: "orange",
+    showIf: (): boolean => true,
+    subcategories: [
+      {
+        name: "basicInformation",
+        label: "Basic Information",
+        fields: [
+          {
+            name: "sector",
+            label: "Sector",
+            description:
+              "Please select the industry sector in which your company was mainly active in the relevant fiscal year.",
+            unit: "",
+            component: "NaceCodeFormField",
+            evidenceDesired: false,
+            required: true,
+            showIf: (): boolean => true,
+            validation: "required|length:1",
+          },
+          {
+            name: "addressOfHeadquarters",
+            label: "Address Of Headquarters",
+            description: "Please provide the full address of your company's headquarters in the relevant fiscal year.",
+            unit: "",
+            component: "AddressFormField",
+            evidenceDesired: false,
+            required: true,
+            showIf: (): boolean => true,
+            validation: "required",
+          },
+          {
+            name: "numberOfEmployees",
+            label: "Number of Employees",
+            description: "Please provide the number of workforce employed by your company in the relevant fiscal year.",
+            unit: "",
+            component: "NumberFormField",
+            evidenceDesired: false,
+            required: true,
+            showIf: (): boolean => true,
+            validation: "required",
+          },
+          {
+            name: "fiscalYearStart",
+            label: "Fiscal Year Start",
+            description: "Please provide the starting date of the company's fiscal year to which you refer.",
+            unit: "",
+            component: "DateFormField",
+            evidenceDesired: false,
+            required: true,
+            showIf: (): boolean => true,
+            validation: "required",
+          },
+        ],
+      },
+      {
+        name: "businessNumbers",
```

Comment:
Should have addressed this in the previous ticket but please rename this section to _Company financials_

Reply:
Done